packages/x-scheduler/src/joy/event-calendar/EventCalendar.tsx

```diff
@@ -5,6 +5,7 @@ import '../index.css';
 import './EventCalendar.css';
```

Comment:
Could you also add the code that renders the right view conditionally? 
With TODOs for the non-implemented views
It could avoid some small Git conflicts if two engineers implement a new view at the same time and add this behavior.